xds/src/main/java/io/grpc/xds/ClientXdsClient.java

```diff
@@ -741,6 +745,10 @@ private static FilterChainMatch parseFilterChainMatch(
       throw new ResourceInvalidException(
           "HttpConnectionManager with xff_num_trusted_hops unsupported");
     }
+    if (enableRbac && !isForClient && !proto.getOriginalIpDetectionExtensionsList().isEmpty()) {
+      throw new ResourceInvalidException("HttpConnectionManager original_ip_detection_extensions "
+          + "must be empty when rbac is enabled.");
```

Comment:
"when rbac is enabled" is confusing, as the rbac filter doesn't have to be present. Just say "unsupported" like the done with `xff_num_trusted_hops`?

(Really, seems the xff_num_trusted_hops check should also have `enableRbac` condition)